README.md

```diff
@@ -3,23 +3,30 @@
 One-line intro: xarray based verification (accuracy) scoring that can scale with Dask if needed. Pandas supported where possible.
 Why use it: trusted implementations, novel metrics, performance, one-stop-shop.
 
+Included Metrics:
+
+| continuous | probability | statistical tests      |
+| ---------- | ----------- | ----------- |
+| MAE, MSE, RMSE, FIRM | CRPS, FIRM |  Diebold Mariano (with the Harvey 1997 and Hering 2011 modifications) |
```

Comment:
It should be "Hering and Genton", not just Hering

Reply:
Now reads Diebold Mariano (with the Harvey et al. 1997 and the Hering and Genton 2011 modifications)